ibis/omniscidb/client.py

```diff
@@ -502,6 +502,10 @@ def _fully_qualified_name(self, name, database):
         # OmniSciDB raises error sometimes with qualified names
         return name
 
+    def _get_list(self, cur):
+        tuples = cur.cursor.fetchall()
+        return list(map(lambda v: v[0], tuples))
```

Comment:
Probably better to write this as `[v[0] for v in tuples]`. It's shorter and probably a tiny bit faster.